notNeededPackages.json

```diff
@@ -404,6 +404,10 @@
             "libraryName": "ci-info",
             "asOfVersion": "3.1.0"
         },
+        "ckeditor": {
+            "libraryName": "@types/ckeditor4",
+            "asOfVersion": "4.16.0"
+        },
```

Comment:
I think you don't need this one. Are you adding a `new` package in this PR, aren't you?
The referenced PR just removed sources (but left published NPM types untouched).

Reply:
well, AFAICT, this would deprecate the old `@types/ckeditor` package and make it load the maintained `@types/ckeditor4` type definitions rather than people keeping usages of the unmaintained `@types/ckeditor` without any warning of it being unmaintained.

Reply:
The source was just removed, IMO current registration serves the purpose (types resolution). 
There is no NPM entry for `ckeditor4` (there is one for `ckeditor4-dev`), how this would help?

Reply:
> There is no NPM entry for `ckeditor4` (there is one for `ckeditor4-dev`), how this would help?

I don't understand what you mean. https://www.npmjs.com/package/ckeditor4 does exist. And if you talk about `@types/ckeditor4`, it will exist once this PR is merged.

Reply:
@stof really sorry for delay. Bear with me, cannot expalin why I was not able to get
ckeditor4 listed on npm (only that dev) 🙇🏻